替换登录主题为theme2的时候，遇到如下错误：

login/login.go:51:28: cannot use Get(cfg...) (type *Login) as type "github.com/GoAdminGroup/go-admin/template".Component in argument to "github.com/GoAdminGroup/go-admin/template".AddLoginComp:
        *Login does not implement "github.com/GoAdminGroup/go-admin/template".Component (missing GetCSS method)

--- a/login/login.go
+++ b/login/login.go
@@ -15,6 +15,7 @@ import (
 	template2 "github.com/GoAdminGroup/go-admin/template"
 	"github.com/GoAdminGroup/go-admin/template/login"
 	"github.com/dchest/captcha"
+	"github.com/GoAdminGroup/go-admin/template/types"
 )
 
 var themes = map[string]Theme{
@@ -130,6 +131,9 @@ func (l *Login) GetAssetList() []string               { return themes[l.Theme].G
 func (l *Login) GetAsset(name string) ([]byte, error) { return themes[l.Theme].GetAsset(name[1:]) }
 func (l *Login) GetName() string                      { return "login" }
 func (l *Login) IsAPage() bool                        { return true }
+func (l *Login) GetJS() template.JS            				{ return "" }
+func (l *Login) GetCSS() template.CSS          				{ return "" }
+func (l *Login) GetCallbacks() types.Callbacks 				{ return make(types.Callbacks, 0) }
 
 func (l *Login) GetContent() template.HTML {
 	buffer := new(bytes.Buffer)

